src/App.css

```diff
@@ -44,3 +44,17 @@ h1 {
 .items-list {
   list-style-type: none;
 }
+
+.sr-only {
```

Comment:
This didn't end up being used, but if you want to keep it in case you choose to use it later that is fine too 😀

Reply:
Yeah, I think it may come in handy later this week as we implement the styling.